### DISCUSSION TITLE
What is the difference between the standard download, and the Legacy versions on the releases page?

### DISCUSSION THREAD

**Discussion by dgnuff** [keepassxreboot/keepassxc#10546] [Q&A] [answered]
The main download button downloads a file called  KeePassXC-2.7.7-Win64.msi .  If I go to the releases page at https://github.com/keepassxreboot/keepassxc/releases I see four options for Windows, all of which have the string "Legacy" in the filename.

A search with Google doesn't turn up anything that explains what the Legacy versions are.

So what is the difference between using the MSI, and downloading the Legacy .zip file to create a portable installation?

**Comment by droidmonkey** [comment, marked as answer]
Legacy is meant for windows 7 and below

![Screenshot_20240404_214429_Edge.png](https://github.com/keepassxreboot/keepassxc/assets/2809491/91902f90-86a6-4790-a756-49b0ef7bcdea)
